File: Java/LeetCode/Number of Employees Who Met the Target.java
// https://leetcode.com/problems/number-of-employees-who-met-the-target

class Solution {

    public int numberOfEmployeesWhoMetTarget(int[] hours, int target) {
        int count = 0;

        for (int h : hours) {
            if (h >= target) {
                count++;
            }
        }

        return count;
    }

    // Alternative solution
    /*
    public int numberOfEmployeesWhoMetTarget(int[] hours, int target) {
        return (int) Arrays.stream(hours)
            .filter((el) -> el >= target)
            .count();
    }
    */
    
}

File: Kotlin/LeetCode/Number of Employees Who Met the Target.kt
// https://leetcode.com/problems/number-of-employees-who-met-the-target

class Solution {

    fun numberOfEmployeesWhoMetTarget(hours: IntArray, target: Int): Int {
        var count = 0;

        for (h in hours) {
            if (h >= target) {
                count++;
            }
        }

        return count;
    }

    // Alternative solution
    /*
    fun numberOfEmployeesWhoMetTarget(hours: IntArray, target: Int) = hours.count { it >= target }
    */

}
